packages/pkg/src/types.ts

```diff
@@ -43,3 +43,7 @@ export interface StringArrayExpression extends ArrayExpression {
 }
 
 export type { StringLiteral } from '@babel/types'
+
+export interface FormatOptions {
+  packageSortOrder?: string[]
```

Comment:
```suggestion
  sortOrder?: string[]
```

Why should it start with `package`? It's redundant IMO. Same as https://github.com/un-ts/prettier/issues/466#issuecomment-2943148230

Reply:
OK, let's keep it as-is.